source/crud/read-operations/cursor.txt

```diff
@@ -0,0 +1,174 @@
+=========================
+Access Data From a Cursor
+=========================
+
+.. default-domain:: mongodb
+
+Read operations that return multiple documents do not immediately return
+all values matching the query. Because a query can potentially match
+very large sets of documents, these operations rely upon an
+object called a :node-docs:`cursor <api/Cursor.html>`
+that fetches documents in batches to reduce both memory consumption and
+network bandwidth usage. Cursors are highly configurable and offer
+multiple interaction paradigms for different use cases.
+
+The following functions directly return cursors:
+
+- ``Collection.find()``
+
+- ``Collection.aggregate()``
+
+- ``Collection.listIndexes()``
+
+- ``Db.aggregate()``
+
+- ``Db.listCollections()``
+
+Other methods, like ``Collection.findOne()`` and ``Collection.watch()``,
+use cursors to generate results but do not directly expose those
+cursors to the method caller.
+
+Cursor Paradigms
+----------------
+
+You can work with cursors using a number of **cursor paradigms**.
+Many use cases only require one document at a time, so many of these
+cursor paradigms take advantage of this by abstracting away network
```

Comment:
Would this still make sense and convey the intended message if you left out "Many use cases only require one document at a time"? I think we should only include general statements like this that are either explained shortly afterwards or otherwise omit them. E.g. sans general statement
"The cursor paradigms allow you to access the query result one document at a time and abstract away the network and caching logic".

Reply:
Reworded this section -- take another look and see if you have any suggestions. Incorporated your suggestions here.